common/config/rush/command-line.json

```diff
@@ -41,6 +51,26 @@
             "summary": "Run all build for all packages",
             "name": "rebuild",
             "allowWarningsInSuccessfulBuild": true
-        }  
+        },
+        {
+            "commandKind": "bulk",
+            "name": "ai-min",
```

Comment:
adding new targets `rush ai-min` and `rush ai-unmin` runs the minification and unminification for all files.

This is necessary for
- When the compile fails you may need to manually run the `unmin` to revert the file back to the original source
- Validating that running multiple times in either direction doesn't corrupt the file.